charts/newrelic-infrastructure-v3/templates/controlplane/daemonset.yaml

```diff
@@ -11,10 +11,14 @@ metadata:
   annotations: {{ . | nindent 4 }}
   {{- end }}
 spec:
-  {{- if eq .Values.controlPlane.kind "DaemonSet"}}
+  {{- if eq .Values.controlPlane.kind "DaemonSet" }}
   {{- with .Values.updateStrategy }}
   updateStrategy: {{ toYaml . | nindent 4 }}
   {{- end }}
+  {{- else if eq .Values.controlPlane.kind "Deployment" }}
+  {{- with .Values.controlPlane.strategy }}
```

Comment:
I still think that if we have ".Values.updateStrategy" we should have ".Values.strategy"

Reply:
Good point, I can change that :)